physics/GFS_MP_generic_post.F90

```diff
@@ -121,6 +134,52 @@ subroutine GFS_MP_generic_post_run(
       do i = 1, im
         rain(i) = rainc(i) + frain * rain1(i) ! time-step convective plus explicit
       enddo
+!
+! Combine convective reflectivity with MP reflectivity for selected
+! parameterizations.
+     if ( (imp_physics==imp_physics_thompson .or. imp_physics==imp_physics_nssl) .and. &
+       (imfdeepcnv==imfdeepcnv_samf .or. imfdeepcnv==imfdeepcnv_gf .or. imfshalcnv==imfshalcnv_gf) ) then
+         do i=1,im
+           factor(i) = 0.0
+           lfrz = .true.
+           zfrz(i) = phil(i,1)*onebg 
+           do k = levs, 1, -1
+             zo(i,k) =  phil(i,k)*onebg 
+             if (gt0(i,k) >= 273.16 .and. lfrz) then
```

Comment:
Any reason not to use one of the following physical constants that are already defined and used elsewhere in the physics?

`[con_t0c]`
 `standard_name = temperature_at_zero_celsius`
  `long_name = temperature at 0 degree Celsius`
  `units = K`
  `dimensions = ()`
  `type = real`
  `kind = kind_phys`
`[con_ttp]`
  `standard_name = triple_point_temperature_of_water`
  `long_name = triple point temperature of water`
  `units = K`
  `dimensions = ()`
  `type = real`
  `kind = kind_phys`

Reply:
triple_point_of_temperature_of_water is defined as 273.16 in physcons.F90

Reply:
> Any reason not to use one of the following physical constants that are already defined and used elsewhere in the physics?
> 
> `[con_t0c]` `standard_name = temperature_at_zero_celsius` `long_name = temperature at 0 degree Celsius` `units = K` `dimensions = ()` `type = real` `kind = kind_phys` `[con_ttp]` `standard_name = triple_point_temperature_of_water` `long_name = triple point temperature of water` `units = K` `dimensions = ()` `type = real` `kind = kind_phys`

Thanks for pointing this out. I made the change.